tree/10_ncf_internals/configuration.cf

```diff
@@ -79,6 +79,18 @@ bundle agent configuration
       "pass2" expression => "pass1";
       "pass1" expression => "any";
 
+      # Make log level incremental
+      "debug"        expression => "trace",
+                          scope => "namespace";
+      # CFEngine's stdlib verbose mode
+      "verbose_mode" expression => "debug",
+                          scope => "namespace";
+      "info"         expression => "debug",
```

Comment:
I've just seen your upper comment.
Don't we need to split debug and trace too in rudder agent ?